sdk/js/src/Client.ts

```diff
@@ -123,6 +123,11 @@ export class DVCClient implements Client {
                     ? EventTypes.variableDefaulted
                     : EventTypes.variableEvaluated,
                 target: variable.key,
+                metaData: {
+                    value: variable.value,
+                    type: typeof(variable.defaultValue),
```

Comment:
this should match the set of types defined here
https://github.com/taplytics/DevCycle/blob/main/lib/shared/mongo/schemas/src/lib/schemas/variable.ts#L120
ie. String, Number, Boolean or JSON